### PR TITLE
Fix kolibri_settings.json when add external plugin

### PR DESCRIPTION
**Pull request by 66eli77**
now conf.py will add all plugins to INSTALLED_APPS

### CODE REVIEW

**Comment by indirectlylit** [comment]
Mind describing the difference between the previous and new behaviors?

**Comment by 66eli77** [comment]
the previous doesn't seem to add plugins to kolibri_settings.json, now it adds the plugins to kolibri_settings.json.

File: kolibri/utils/conf.py [hunk 1]
```diff
@@ -54,7 +48,8 @@
     "kolibri.plugins.video_mp4_render",
     "kolibri.plugins.audio_mp3_render",
     "kolibri.plugins.setup_wizard",
-    "kolibri.plugins.coach_tools"
+    "kolibri.plugins.coach_tools",
+    "exercise_perseus_render"
```
Comment (indirectlylit) [inline kolibri/utils/conf.py:52]: how hard would it be to change the name to `exercise_perseus_renderer`?

Comment (jamalex) [reply kolibri/utils/conf.py:52]: `render` looks funny to me too, but note the names of the audio/video/document ones above, too

File: kolibri/utils/conf.py [hunk 2]
```diff
@@ -64,9 +59,16 @@
...
 
-# Open up the config file and overwrite defaults
-with open(conf_file, 'r') as kolibri_conf_file:
-    config.update(json.load(kolibri_conf_file))
+conf_file = os.path.join(KOLIBRI_HOME, "kolibri_settings.json")
+
+if not os.path.isfile(conf_file):
+    logger.info("Initialize kolibri_settings.json..")
+    with open(conf_file, 'w') as kolibri_conf_file:
+        json.dump(config, kolibri_conf_file)
```
Comment (jamalex) [inline kolibri/utils/conf.py:68]: Looks like these two lines of code are identical to the contents of the `save` function -- could you call that instead?

File: kolibri/utils/conf.py (hunk 1 above)
Comment (indirectlylit) [reply kolibri/utils/conf.py:52]: oh yeah. well, better to be consistently wrong then :)

File: kolibri/utils/conf.py (hunk 2 above)
Comment (66eli77) [inline kolibri/utils/conf.py]: it's a bit tricky to use save here because save's state depends on the outcome of this if..else statement.

**Pull request merged**